sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/AutoConfiguredOpenTelemetrySdkBuilder.java

```diff
@@ -541,13 +541,7 @@ private static AutoConfiguredOpenTelemetrySdk maybeConfigureFromFile(
       return null;
     }
     logger.fine("Autoconfiguring from configuration file: " + configurationFile);
-    FileInputStream fis;
-    try {
-      fis = new FileInputStream(configurationFile);
-    } catch (FileNotFoundException e) {
-      throw new ConfigurationException("Configuration file not found", e);
-    }
-    try {
+    try (FileInputStream fis = new FileInputStream(configurationFile)) {
```

Comment:
Nice catch here to close this. Is is really the case that `IOException` is only thrown when an error occurs closing the file?

Reply:
yeah, I was kinda surprised by that too, I'll add a comment in that catch block